recipes/stock-pandas/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "1.4.5" %}
+
+package:
+  name: stock-pandas
+  version: {{ version }}
+
+source:
+  url: https://github.com/kaelzhang/stock-pandas/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ec8f3c97fe4a1b60541fc3b059f8fcfa1c288669b0a9e6e8eb27730dac9c4e01
+
+build:
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+  - STOCK_PANDAS_BUILDING_EXT=1
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - Cython >=3.0.5,<4.0.0
+    - numpy >=2.0.0rc1
```

Please have an exact pin here. Once you are on the feedstock, you should move this to a plain `numpy` entry so that the global numpy pin is used (the global is currently at 1.26x, but there is a migration the bot will attempt to move each feedstock to 2.x)
```suggestion
    - numpy 2.0
```